addons/web_editor/tests/test_tools.py

```diff
@@ -66,14 +66,19 @@ def test_valid_video_url(self):
 
 @tagged('-standard', 'external')
 class TestVideoUtilsExternal(common.BaseCase):
-    def test_get_video_thumbnail(self):
+    def test_get_video_metadata(self):
         #youtube
-        self.assertIsInstance(tools.get_video_thumbnail(TestVideoUtils.urls['youtube']), bytes)
+        self.assertIsInstance(tools.get_video_metadata(TestVideoUtils.urls['youtube'])["thumbnail"], bytes)
+        self.assertIsInstance(tools.get_video_metadata(TestVideoUtils.urls['youtube'])["name"], str)
```

Comment:
Even if platforms don't have a rate limiting, maybe it's not necessary to call each of them twice just to check two properties which should not change between calls?

Especially when the new call seems useless, even if the fetch fails (and thus the previous call already blows up) the name gets set to a string.